app/views/admin/loans/show.html.slim

```diff
@@ -36,6 +34,12 @@ section.loan.details class=(@loan.valid? ? 'show-view' : 'edit-view')
 
   div.loan-content
     = render "admin/loans/tabs/tabs"
-
-  javascript:
-    $(function() { new MS.Views.ShowEditView({ el: '.loan.details' }) });
+    
+javascript:
+  $(function(){
+    new MS.Views.ShowEditView({ el: '.loan.details' })
+    new MS.Views.LoanTabView({
+      el: '.nav-tabs',
+      calEvents: #{@calEvents.to_json.html_safe}
```

Comment:
Yes, that's exactly why we _don't_ want to use `html_safe`. I guess that's kind of counterintuitive! But html_safe is _declaring_ that the string has no bad html inside it. If we can't be sure of that, we can't use html_safe.